### ISSUE TITLE
platform.api.post("user/session",param) is broken after updating from DF 2.0.1 -> DF 2.0.4

### ISSUE THREAD

**Issue by bitperbit** [dreamfactorysoftware/dreamfactory#26]
Hi, 

After updating to DF 2.0.4 the server side scripts will fail when calling platform.api.post("user/session",{}).

How to replicate: I created a service test1 that will execute this:

```javascript
var param = {"email": "test1@bitperbit.info", "password": "test1","remember_me": true, "duration": 64000};
var res_login = platform.api.post("user/session",param);
event.response = res_login ;
return;
```

This will return an 500 Internal Server Error, with this message: 
```html
<span class="exception_title">
	<abbr title="Symfony\Component\Debug\Exception\FatalErrorException">FatalErrorException</abbr> in 
	<a title="/home/bitnami/dreamfactory-2.0.1-0/apps/dreamfactory/htdocs/app/Http/Middleware/AuthCheck.php line 29" ondblclick="var f=this.innerHTML;this.innerHTML=this.title;this.title=f;">AuthCheck.php line 29</a>:
</span>
<span class="exception_message">Call to a member function query() on a non-object</span>

</h2>
<div class="block">
<ol class="traces list_exception">
	<li> in 
		<a title="/home/bitnami/dreamfactory-2.0.1-0/apps/dreamfactory/htdocs/app/Http/Middleware/AuthCheck.php line 29" ondblclick="var f=this.innerHTML;this.innerHTML=this.title;this.title=f;">AuthCheck.php line 29</a>
	</li>

</ol>
</div>
```

This i happening only on custom serverside script. If you make direct calls to "api/v2/user/session" with the API Docs or via Postman it works. It seems to be broken only on scripts.

Please check and advise when this will be fixed.
Best regards
Gabriel

**Comment by andersonmuller** [comment]
Same problem here, I tested in versions 2.02 and 2.04

**Comment by drewpearce** [comment]
Confirmed bug. DF-593. Filed with dev team.

**Comment by leehicks** [comment]
Fixed in dreamfactory/df-core 0.2.1, released in dreamfactory/dreamfactory installs for 2.1.0.
